src/core/handle.ml

```diff
@@ -268,12 +278,13 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
     indicate commands that take too long to execute. *)
 let too_long = Stdlib.ref infinity
 
-(** [handle_cmd ss cmd] is similar to [handle_cmd_aux ss cmd] but it adds some
+(** [handle_cmd ss cmd] is similar to [handle_cmd ss cmd] but it adds some
```

Comment:
Surely `handle_cmd ss cmd` is similar to `handle_cmd ss cmd`. Did the comment make sense before or was that not up to date already?